android/app/src/main/java/org/fossasia/openevent/core/feed/facebook/FacebookFeedFragment.java

```diff
@@ -63,8 +61,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         facebookFeedFragmentViewModel = ViewModelProviders.of(this).get(FacebookFeedFragmentViewModel.class);
         RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(getContext());
         feedRecyclerView.setLayoutManager(mLayoutManager);
-        facebookFeedAdapter = new FacebookFeedAdapter(getContext(), openCommentsDialogListener, feedItems);
-        facebookFeedAdapter.setOnImageZoomListener(onImageZoomListener);
+        facebookFeedAdapter = new FacebookFeedAdapter(getContext(), this, feedItems);
+        facebookFeedAdapter.setOnImageZoomListener(this);
```

Comment:
Similar to zoom listener, add the comment dialog listener through a setter, and remove it on onDestroy